dcbr-web/src/components/AnimalIdentification.vue

```diff
@@ -82,7 +82,7 @@ export default {
         if (value === "false") {
           this.$store.dispatch(
             "animalIdentification/permIdType",
-            ""
+            "NOT_APPLICABLE"
```

Comment:
Is this meant to be a string, or are we referencing a variable here?

Reply:
This is intended to be a string. The API expects either MICROCHIP, TATTOO, OTHER or NOT_APPLICABLE. It submits NOT_APPLICABLE when the previous question is answered false, as we cannot submit an empty string to the API for this field.